src/Components/IndexList/IndexList.scss

```diff
@@ -154,7 +154,6 @@ $cf-index-list--sort-text: $g11-sidewalk;
 
 // Empty state
 .cf-index-list--empty-cell {
-  background-color: $cf-card-text--disabled;
```

Comment:
I think this was supposed to be `$cf-card-background--disabled`

Reply:
@mavarius good catch

Reply:
Thanks guys! I have fixed it. Screenshot updated as well.